### DISCUSSION TITLE
Some Smart NORA devices do not respond to voice commands in Google Home

### DISCUSSION THREAD

**Discussion by Anjerlaan** [andrei-tatar/node-red-contrib-smartnora#276] [General]
Hi,

After my holiday return, 2 (of 5) Smart NORA Lights do not respond to their voice command anymore.
However, If I say All Lights ON/OF, they work.
The ON/OFF buttons of those devices in the Google Home App do respond normally.
Does anyone experience the same issue?

**Comment by Anjerlaan** [comment]
Okay my other Google Home Mini's are working normally.....so there is something going on with that google home mini device.
Already restarted but did not solve the issue

**Reply by andrei-tatar** [reply]
Hmm, I couldn't say what the issue is. Do you have multiple WiFi networks with NAT in your home? Or are all the devices in the same network?
